### PR TITLE
Adding Gatsby Docs

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0",
     "@fluentui/react-native": "^0.14.4",
     "@mdx-js/loader": "^1.5.5",
     "@mdx-js/mdx": "^1.5.5",
     "@next/mdx": "^9.2.1",
-    "next": "^9.2.1",
-    "react": "16.8.6",
-    "react-dom": "16.8.6"
+    "next": "^9.2.1"
```
Comment (PPatBoyd) [inline docs/package.json:33]: we can remove next dependency

**Comment by micahgodbolt** [comment]
@PPatBoyd everything looks good

**Comment by micahgodbolt** [comment]
@PPatBoyd if you want to add me to the project, happy to merge

**Comment by PPatBoyd** [comment]
> @PPatBoyd if you want to add me to the project, happy to merge

it's controlled by office dev and/or office cxe org membership -- LMK if you have issues joining, I'll merge this one nbd :)

**Pull request merged**